proto/ssn/type/candidate.proto

```diff
@@ -63,3 +63,15 @@ message Candidate {
   // page_ref start from 1.
   uint32 page_ref = 6;
 }
+
+message LineCandidate {
```

Comment:
Claus' request that they should be able to extract the actual value of the line? 
Or do they just want to know what the line said? (In that case we can just use a normal candidate)

Reply:
Yes, the request from claus was to have price and line text separated